docs/studio/cli/README.md

```diff
@@ -65,6 +81,7 @@ You can see all of these commands available by either running the `help` command
 `studio project <subcommand>`: Manage your studio projects.
 
 - `ls [team_id]`: List the projects for the given ID, or if no id is given, use the default context.
+- `create <name> <description>`: Creates a new project with the given name and description, also requiring the `--teamId` flag is passed.
```

Comment:
The requirement for the `--teamId` flag was removed, but it may not have been published yet.
@dtbuchholz if you're ok with it, we can remove the note from the docs and i'll publish the newest version before EOW.

Reply:
got it, yeah i get an error with the latest release w/o the team ID flag: `Error: must provide team for project`. but yeah, i'll update/remove that in my new docs updates and can push it once you publish the new version.